crowbar_framework/config/locales/ceph/en.yml

```diff
@@ -43,3 +43,5 @@ en:
             enabled: 'Protocol'
             certfile: 'SSL Certificate File'
             keyfile: 'SSL (Private) Key File'
+      validation:
+        ses_repos: 'The SUSE Enterprise Storage repositories have not been setup.'
```

Comment:
So, it's not introduced by your commit, but I think the error string should be made vendor-neutral. Something like "Ceph is not available in the repositories.", and then we would patch the string in the package to mention SES.